### PR TITLE
Feature/separate adapters module

### PR DESCRIPTION
**Pull request by Skejven**
Separate adapters into separate root module
---

I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

**Comment by wiiitek** [comment]
Hello, I see there are a lot of changes withn Knotx lately :)
As there are new modules created, could we add a REDME.md file for every module that contain:

- one sentence about what is the module for
- one paragraph with more detailed description (less than 50 words)

this would:
* make it easier for new people to understand the modules...
* help us maintain the scope/responsibility of modules

Cheers :)

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ You will see output similar to the following:
 		Deployed fe2a7ef0-1f91-4ee0-ac7a-8617aedd8f9a [com.cognifide.knotx.mocks.MockServiceAdapterVerticle]
 		Deployed 1d92984b-27a2-495d-baa1-50bea688c869 [com.cognifide.knotx.repository.FilesystemRepositoryVerticle]
 		Deployed 11776e93-4541-465a-ab05-a0bf472088a6 [com.cognifide.knotx.engine.view.ViewEngineVerticle]
-		Deployed 73e3ad21-b121-4d13-8cd4-c17ea868a640 [com.cognifide.knotx.core.serviceadapter.http.HttpServiceAdapterVerticle]
+		Deployed 73e3ad21-b121-4d13-8cd4-c17ea868a640 [com.cognifide.knotx.adapter.service.http.HttpActionAdapterVerticle]
```
Comment (tomaszmichalak) [inline README.md]: I think that two adapters are started: action and service adapter.

**Comment by tomaszmichalak** [comment]
@wiiitek we are working on major changes in knotx. we will deliver documentation after base implementation.

**Comment by Skejven** [comment]
@wiiitek very good idea, I've created ticket #128 to not forget about it after we finish implementation. Thank you!

**Pull request merged**